backbone.stickit.js

```diff
@@ -501,34 +513,54 @@
       // If there are no `selectOptions` then we assume that the `<select>`
       // is pre-rendered and that we need to generate the collection.
       if (!selectConfig) {
-        selectConfig = {};
-        var getList = function($el) {
-          return $el.map(function(index, option) {
-            // Retrieve the text and value of the option, preferring "stickit-bind-val"
-            // data attribute over value property.
-            var dataVal = Backbone.$(option).data('stickit-bind-val');
-            return {
-              value: dataVal !== undefined ? dataVal : option.value,
-              label: option.text
-            };
-          }).get();
+        // simple way
+        $el.val(val);
+        return;
+
+        // I do not know if it is possible that this method might be invoked
+        // each time with different params. I |selectConfig| might be present
+        // in future calls after one call with it,
+        // then next code should be enable
+
+        // I left it here so you can check. If that case is not possible
+        // then I will remove this code. If it's possible, then I will enable it
+
+        var check = function(val) {
+          return _.isArray(val) || _.isObject(val);
```

Comment:
If `_.isArray(val)` passes `_.isObject(val)` will pass as well. Check can just be `_.isObject`